inc/RestCommand.php

```diff
@@ -61,9 +61,15 @@ public function set_auth( $auth ) {
 	public function create_item( $args, $assoc_args ) {
 		list( $status, $body ) = $this->do_request( 'POST', $this->get_base_route(), $assoc_args );
 		if ( Utils\get_flag_value( $assoc_args, 'porcelain' ) ) {
-			WP_CLI::line( $body['id'] );
+			if ( $status < 400 ) {
```

Comment:
> Caveat, I am learning Travis on the fly. Might see several commits as I figure out how to use it.

You should be running the tests locally. Here's the appropriate documentation: https://make.wordpress.org/cli/handbook/pull-requests/

> But...what do you think. What is the approach/spec on this? For both porcelain and without?

Porcelain is meant to only ever output the result of the operation that can be piped to another command. I think we should continue with this behavior. If RESTful WP-CLI sometimes doesn't return output, the exit code should indicate whether it was an error or not.

> Would you please add apache to the travis config? It would likely much churn for me to get that working.

You're welcome to work on it in a separate pull request. I don't have the bandwidth to handle this immediately.